### ISSUE TITLE
Asp.Net core razor page errors on text

### ISSUE THREAD

**Issue by McVanS** [JosefPihrt/Roslynator#81]
```html
<header>
    <h2>Access error</h2>
    <hr />
</header>

<div>
    <span>
        Access to the specific resource has been denied!
    </span>
    <span>
        Try logging in first.
    </span>
</div>
```

Causes six RCS1170 Use read-only auto-implemented property.	...\Views\Errors\AccessDeniedError.cshtml	1	Active

As you can see it is just plain text in HTML. I'm pretty sure it should not cause any errors.

Microsoft Visual Studio Community 2017 
Version 15.1 (26403.3) Release
VisualStudio.15.Release/15.1.0+26403.3
Microsoft .NET Framework
Version 4.6.01590

Installed Version: Community

Visual Basic 2017   00369-60000-00001-AA085
Microsoft Visual Basic 2017

Visual C# 2017   00369-60000-00001-AA085
Microsoft Visual C# 2017

ASP.NET and Web Tools 2017   15.0.30320.0
ASP.NET and Web Tools 2017

ASP.NET Web Frameworks and Tools 2017   5.2.50303.0
For additional information, visit https://www.asp.net/

Azure App Service Tools v3.0.0   15.0.30209.0
Azure App Service Tools v3.0.0

Azure Data Lake Node   1.0
This package contains the Data Lake integration nodes for Server Explorer.

Azure Data Lake Tools for Visual Studio   2.2.5000.0
Microsoft Azure Data Lake Tools for Visual Studio

Bundler & Minifier   2.4.340
Adds support for bundling and minifying JavaScript, CSS and HTML files in any project.

Common Azure Tools   1.9
Provides common services for use by Azure Mobile Services and Microsoft Azure Tools.

Fabric.DiagnosticEvents   1.0
Fabric Diagnostic Events

JavaScript Language Service   2.0
JavaScript Language Service

KofePackagePackage Extension   1.0
KofePackagePackage Visual Studio Extension Detailed Info

Microsoft Azure Hive Query Language Service   2.2.5000.0
Language service for Hive query

Microsoft Azure Tools   2.9
Microsoft Azure Tools for Microsoft Visual Studio 2017 - v2.9.50131.1

Microsoft MI-Based Debugger   1.0
Provides support for connecting Visual Studio to MI compatible debuggers

NuGet Package Manager   4.1.0
NuGet Package Manager in Visual Studio. For more information about NuGet, visit http://docs.nuget.org/.

Redgate SQL Search   2.4.1.1284
Search functionality for SQL Server databases, from within Visual Studio

SQL Server Data Tools   15.1.61702.140
Microsoft SQL Server Data Tools

ToolWindowHostedEditor   1.0
Hosting json editor into a tool window

TypeScript   2.1.5.0
TypeScript tools for Visual Studio

Visual Studio Spell Check Everywhere   VSSpellCheckEverywhere
An extension that enables spell checking within any Visual Studio file editor or tool window that uses WPF text boxes.
https://GitHub.com/EWSoftware/VSSpellChecker

Visual Studio Spell Checker   VSSpellChecker
An editor extension that checks the spelling of comments, strings, and plain text as you type or interactively with tool windows.
https://GitHub.com/EWSoftware/VSSpellChecker

Web Compiler   1.11.326
Compiler for LESS, Sass and CoffeeScript files

P.S. Love your work. Keep it up.

**Comment by JosefPihrt** [comment]
These diagnostics are triggered by auto-generated code. But it seems that this code is not marked as auto-generated so it cannot be skipped by roslyn.

I created relevant [issue](https://github.com/aspnet/Razor/issues/1215) in aspnet/Razor repo.

**Comment by JosefPihrt** [comment]
There is a workaround, but it's not nice.

```csharp
[assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("", "RCS1170", Scope = "member", Target = "~P:AspNetCore._Page_Views_Home_About_cshtml.ModelExpressionProvider")]
[assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("", "RCS1170", Scope = "member", Target = "~P:AspNetCore._Page_Views_Home_About_cshtml.Url")]
[assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("", "RCS1170", Scope = "member", Target = "~P:AspNetCore._Page_Views_Home_About_cshtml.Component")]
[assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("", "RCS1170", Scope = "member", Target = "~P:AspNetCore._Page_Views_Home_About_cshtml.Json")]
[assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("", "RCS1170", Scope = "member", Target = "~P:AspNetCore._Page_Views_Home_About_cshtml.Html")]
```

**Comment by JosefPihrt** [comment]
It's being tracked [here](http://github.com/aspnet/Mvc/issues/5957).
